.gitignore

```diff
@@ -0,0 +1,33 @@
+HELP.md
+target/
+!.mvn/wrapper/maven-wrapper.jar
```

Comment:
remove .mvn and sub folders from git

Reply:
gitignore updated to ignore maven files/folder